core/kits/manager.php

```diff
@@ -40,9 +40,9 @@ public function get_active_kit() {
 	public function get_active_kit_for_frontend() {
 		$kit = Plugin::$instance->documents->get_doc_for_frontend( $this->get_active_id() );
 
-		if ( ! $this->is_valid_kit( $kit ) ) {
-			return $this->get_empty_kit_instance();
-		}
+		// if ( ! $this->is_valid_kit( $kit ) ) {
+		// 	return $this->get_empty_kit_instance();
+		// }
```

Comment:
What is this? @hein-obox 
I think you might have gotten mixed up with Louis on this one.

Reply:
* Sorry, I accidentely merged some other files while working late *